identity/src/main/java/com/android/identity/trustmanagement/CustomValidators.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android.identity.trustmanagement
+
+import java.security.cert.PKIXCertPathChecker
+
+/**
+ * Object used to obtain custom validators based on docType
+ */
+object CustomValidators {
+    fun getByDocType(docType: String): List<PKIXCertPathChecker>
+    {
+        when (docType){
+            "org.iso.18013.5.1.mDL" -> return listOf(CountryValidator())
+            else -> return emptyList()
+        }
+    }
+}
```

Comment:
Suggest to push this helper to the reader app... having it in the library creates the risk that the app assumes the library knows what is right for a doc-type. Much rather have the app be explicit about what it needs.

Reply:
Ok, moved the CountryValidator and the CustomValidators to the reader app